test/team_test.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/team.rb'
+require './lib/player.rb'
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+
+class TeamTest < Minitest::Test
+
+  def test_it_exists
+    france = Team.new("France")
+    assert_instance_of Team, france
+  end
+
+  def test_it_has_country
+    france = Team.new("France")
+    assert_equal "France", france.country
+  end
+
+  def test_it_starts_with_no_players
+    france = Team.new("France")
+    assert_equal [], france.players
+  end
+
+  def test_it_starts_with_false_eliminated_status
+    france = Team.new("France")
+    refute france.eliminated?
+  end
+
+  def test_it_can_check_eliminated_status
+    france = Team.new("France")
+    assert france.eliminated = true
+  end
+
+  def test_it_can_change_eliminated_status
+    france = Team.new("France")
+    france.eliminated = true
```

Comment:
Be super careful reading the spec!  a Team should have an attribute of `@eliminated`, but the method that we want to call is `eliminated?`